src/dms-preview/azext_dms/_help.py

```diff
@@ -5,6 +5,100 @@
+"""
+
+helps['dms task create'] = """
+    type: command
+    short-summary: Create and start a service-level task.
+    long-summary: |
+        The following tasks are currently supported:
+            1) Check OCI driver against Oracle source to check for compatibility
+            2) Upload OCI driver install package
+            3) Install an already uploaded OCI driver package
+
+    parameters:
+        - name: --task-type
+          type: string
+          short-summary: >
+            The type of task to create. The supported types are CheckOciDriver, UploadOciDriver, and InstallOciDriver.
+        - name: --task-options-json
+          type: string
+          short-summary: >
+            Information and settings relevant to the particular task's type. This can be either a JSON-formatted string or the location to a file containing the JSON object. See examples below for the format.
+          long-summary: >
+            For CheckOciDriver:
+                {
+                    // The version of your Oracle source server against which to check the driver's compatibility.
+                    // Can also be null to get a list of supported versions.
+                    "serverVersion": "Oracle source version"
+                }
+
+            For UploadOciDriver:
+                {
+                    "ociDriverPath": File share path of the driver install package,
+                    "userName": "user name",    // if this is missing or null, you will be prompted
+                    "password": "password"      // if this is missing or null (highly recommended) you will be prompted
+                }
+
+            For InstallOciDriver:
+                {
+                    // The output of the UploadOciDriver task can be used here
+                    "ociDriverPackageName": "Oci driver installer package name"
+                }
+    examples:
+        - name: Create a task which checks if the currently installed OCI driver supports
+          text: >
+            az dms task create -g myresourcegroup --service-name mydms -n checkocitask1 --task-options-json "{\"serverVersion\": \"12.2.0.1\"}" --task-type CheckOciDriver
+"""
+
+helps['dms task delete'] = """
+    type: command
+    short-summary: Delete a service-level Task.
+    parameters:
+        - name: --delete-running-tasks
+          type: bool
+          short-summary: >
+            Delete the service-level Task even if the Task is currently running.
+"""
+
+helps['dms task list'] = """
+    type: command
+    short-summary: List the service-level Tasks within a DMS instance. Some tasks may have a status of Unknown, which indicates that an error occurred while querying the status of that task.
+    parameters:
+        - name: --task-type
+          type: string
+          short-summary: >
+            Filters the list by the type of task. For the list of possible types see "az dms check-status".
```

Comment:
For the parameter message, can we use noun value here, e.g. "The type of task to be listed"?